index.Rmd

```diff
@@ -1,105 +1,228 @@
 
-# Results
+The formula is as below:
+[MNDWI=(Green-MIR)/ (Green+MIR)]
+MNDWI <- (image[[2]] - image[[3]])/(image[[2]] + image[[3]]).
 
-[~200 words]
+To plot the results with ggplot, we convert the raster into a data frame and use geom_tile.
+The plot generated by the codes below show where are located water/ water ice/ volatiles materials the most and we have then know where to land the rovers and start the mining process.
 
-Tables and figures (maps and other graphics) are carefully planned to convey the results of your analysis. Intense exploration and evidence of many trials and failures. The author looked at the data in many different ways before coming to the final presentation of the data.
+Band4, Band5, and Band6 are use for Earth datas where Band4 represent the Green band in our case and Band6 represent the MIR band.
 
-Show tables, plots, etc. and describe them.
+For Mars,the formula change little bit because all bands are collected and stored in the same file in Mars Odysee website. False color or black and white could reveal the distribution of water. However, we use the formula:
+
+MNDWI <- (band4)/(1/band9).
+This formula is derived after multiple simulations and observations
+
+```{r}
 
-```{r, fig.width=6, fig.height=3, fig.cap="Map of completely random data"}
-m <- leaflet(data) %>% 
-  addTiles() %>% 
-  addCircleMarkers(~x, ~y, radius = ~size,color = ~as.factor(category)) %>% 
-  addPopups(~x[2], ~y[2], "Random popup")
-m  # a map with the default OSM tile layer
+#We derived This formula is derived after multiple observations and applied to different data before using it.
+MNDWI <- (band4)/(1/band9)
+#MNDWI <- (band4-band9)/(band4+band9)
+
+
+as(MNDWI, "SpatialPixelsDataFrame") %>% 
+  as.data.frame() %>%
+  ggplot(data = .) +
+  geom_tile(aes(x = x, y = y, fill = layer)) +
+  theme(axis.text = element_blank(),
+        axis.ticks = element_blank(),
+        panel.background = element_blank(),
+        panel.grid.minor = element_blank()) +
+  labs(title = "Water or Water ice or Volatiles materials", 
+       x = " ", 
+       y = " ") +
+  scale_fill_gradient(high = "#CEE50E", 
+                      low = "#087F28",
+                      name = "Water or Water ice or Volatiles materials")
+```
+
+## PART2: Visualization of mining sites within the selected landing site on Martian gloge (map) 
+
+After knowing that Lyot crater"V00823007" site is the best site to be consider as best future landing site according to the figure above, We just get the GPS coordinates of some locations within the Lyot crater"V00823007" site  (GPS coordinates from the onboard GPS of the spacecraft), created an exel file containing the coordinates of those locations and plot them on Martian georeferenced globe using the threejs package.
```

Comment:
How do you know it's the 'best'?  Not from this analysis.  So this is more of an introduction than a result.  Please revise.